### ISSUE TITLE
does this support processing 2.1.2?

### ISSUE THREAD

**Issue by lisongx** [TACC/MassivePixelEnvironment#18]
can't running correctly on my machine, anyone still using this recently?

**Comment by lisongx** [comment]
in the `run.sh` file, the app path seems to be 
`./application.macosx/MPEPeasy.app/Contents/MacOS/MPEPeasy` now.
but that can't make it run..

**Comment by rturknett** [comment]
We will be releasing a new version soon that supports 2.1.2. The working source for that version is here, along with the updated MPEPeasy example, but the documentation isn't updated yet:

https://github.com/rturknett/MassivePixelEnvironment/tree/auto_play

**Comment by lisongx** [comment]
@rturknett 
tha'ts great!

BTW, How do I build the processing library from source?

**Comment by rturknett** [comment]
You will need Eclipse and the latest Oracle JDK, and you will need to edit the `resources/build.properties` file for your environment.

Generic instructions for building Processing libraries can be found here:

https://github.com/processing/processing-library-template/blob/master/README.md

**Comment by rturknett** [comment]
I can also send a recent build to your email address if you like.

**Comment by lisongx** [comment]
@rturknett 
it's very kind of you, my email is iseansay@gmail.com. thx!

**Comment by rturknett** [comment]
Hi @seansay, I haven't forgotten about sending you the build. However, we need to update the documentation first since a number of things have changed. Hope to do that early next week. Also, what platform will you be running on?

**Comment by lisongx** [comment]
@rturknett 
I'm on mac os x. thx

**Comment by lisongx** [comment]
@rturknett 
did you have a working example...I'm trying to using shiffman's most pixal ever, but ... it's very frustrating

**Comment by rturknett** [comment]
@seansay 
I can send you a build that includes a working example that we've tested on Linux (CentOS), but we haven't yet documented how to run it.

What kind of cluster are you running on? Are the cluster machines also Mac OSX?

**Comment by lisongx** [comment]
@rturknett 
I was just trying to use this in 3 computers, is this not the right way to achive this?

Basicly I have a processing sketch, and I want to use 3 projector to projecto to three wall in one room.  What do you recommend?

**Comment by rturknett** [comment]
@seansay 
Yes, you can use our MPE library for this. However, for a three projector setup MPE is designed to use 4 computers -- one for the controller/head display and one for each projector. If you have another computer+monitor that you can use for the head display, that would be ideal.

**Comment by lisongx** [comment]
@rturknett 
Currently I'm trying to use mostpixelever, which I can run the server on one of the 3 computers. The problem I have is the sketch just randomly frozen...seems that the client have connection problem with server, do you have any idea why this might be?

**Comment by rturknett** [comment]
Hi @seansay, I can't help you with mostpixelsever, but it should be possible in theory to configure Massive Pixel Environment with the head node (server) on one of the 3 display nodes. We haven't tested that recently, though.
